docs/admin_api/shutdown_room.md

```diff
@@ -79,13 +79,20 @@ Response:
 the structure can and does change without notice.
 
 First, it's important to understand that a room shutdown is very destructive. Undoing a shutdown is not as simple as pretending it
-never happened - work has to be done to move forward instead of resetting the past.
+never happened - work has to be done to move forward instead of resetting the past. In fact, in some cases it might not be possible
+to recover at all:
 
-1. For safety reasons, it is recommended to shut down Synapse prior to continuing.
+* If the room was invite-only, your users will need to be re-invited.
+* If the room no longer has any members at all, it'll be impossible to rejoin.
+  * The first user to rejoin will have to do so via an alias on a different server.
```

Comment:
is the change in list level deliberate?

Reply:
nope, I ran an autoformatter after applying suggestions and it decided that 1 space meant 2. Will push it back